retest.gemspec

```diff
@@ -28,4 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "string-similarity", ["~> 2.1"]
   spec.add_runtime_dependency "listen", ["~> 3.9"]
   spec.add_runtime_dependency "tty-option", ["~> 0.1"]
+  spec.add_runtime_dependency "observer", ["~> 0.1.0"]
```

Comment:
I'm happy to introduce minor version updates from the observer gem. 

```ruby
  spec.add_runtime_dependency "observer", ["~> 0.1"]
```
Can you apply that change and bundle again, please?

I've made a PR that applies the change https://github.com/AaronRustad/retest/pull/1, but feel free to make your own commit on your branch. The PR is mostly to show where all the changes should be applied.

Reply:
Sure thing. Done.